controllers/snapshot/snapshot_adapter.go

```diff
@@ -366,6 +369,29 @@ func (a *Adapter) EnsureGlobalCandidateImageUpdated() (controller.OperationResul
 	return controller.ContinueProcessing()
 }
 
+func GetStatusCondition(snapshot *applicationapiv1alpha1.Snapshot) (condition *metav1.Condition, ok bool) {
+	const (
```

Comment:
Could [these constants](https://github.com/redhat-appstudio/integration-service/blob/02c829368d912e684b82dbab204b34f56e576fba/gitops/snapshot.go#L86-L90) be used instead of defining them here?